common/source/docs/common-buttons.rst

```diff
@@ -19,8 +19,8 @@ Alternatively, a PWM signal can be used on the pin for switching, just like the
 
 There is no MIDDLE position for the function, if it has one, using a button.
 
-The effective state of the button can be inverted by setting its ``BTN_OPTIONSx<BTN_OPTIONSx>`` parameter bit 1. This applies to both PWM and switch control of the pin.
+The effective state of the button can be inverted by setting its ``BTN_OPTIONSx<BTN_OPTIONSx>`` parameter bit 1 (e.g. BTN_OPTIONSx = 2). This applies to both PWM and switch control of the pin.
```

Comment:
this is incorrect....since its a bit mask, the VALUE should have +2 added to it, not set to 2 which possibly destroys the setting of bit 0..the best clarification would be :
```suggestion
The effective state of the button can be inverted by setting its ``BTN_OPTIONSx<BTN_OPTIONSx>``  bitmask bit 1 (ie adding 2 to the decimal value of the parameter). This applies to both PWM and switch control of the pin.
```